cypress/e2e/signup-api.cy.js

```diff
@@ -0,0 +1,102 @@
+//<reference types="cypress" />
```

Comment:
add another / here, right now its only two //.
it should be 3 ///

Reply:
Done